peachjam_search/views.py

```diff
@@ -55,6 +55,17 @@ def construct_search(self, request, view, search_backend):
         ]
 
 
+class RankFeatureBackend(BaseSearchQueryBackend):
+    @classmethod
+    def construct_search(cls, request, view, search_backend):
+        rank = Q(
```

Comment:
Can you simply not return anything if the setting's boost value isn't set? Because even with zero, this will break on indexes that don't yet have the ranking feature.

Reply:
changed